src/Table/ValueTableBuilder.php

```diff
@@ -84,7 +84,7 @@ public function onQuerying(Builder $query)
             $query->where($table . '.entry_id', $entry->getId());
             $query->orderBy($table . '.sort_order', 'ASC');
         } else {
-            $query->whereIn('id', $uploaded ?: [0]);
+            $query->whereIn($query->getModel()->getTable() . '.id', $uploaded ?: [0]);
```

Comment:
This should be the same as `$table` above right? Can you update to use that instead of the additional logic? Thanks!